dotcom-rendering/src/web/components/Page.tsx

```diff
@@ -61,6 +62,13 @@ export const Page = ({ CAPI, NAV, format }: Props) => {
 					isDev={CAPI.config.isDev}
 				/>
 			</Island>
+			<Island clientOnly={true} deferUntil="idle">
```

Comment:
Deferring until idle seems 👌

Reply:
I think it’s all reading [from the `Performance` API](https://developer.mozilla.org/en-US/docs/Web/API/Performance). Indeed if it’s not available you’d need polyfill.

Reply:
Thanks both!
Testing in CODE, initialising the listeners slightly later doesn't seem to impact our ability to collect CWV metics.
I'll monitor after deploying to make sure we don't see a dip in the number of pageviews with this data.